native-installation-instructions.md

```diff
@@ -16,19 +16,19 @@ These are the original instructions for natively installing the app to your mach
 
 ## Set up a Ruby Environment
 
-You will need to install Ruby 2.4.2 using RVM or rbenv.
+You will need to install Ruby 2.7.2 using RVM or rbenv.
 
 ### Option 1: Using [rvm](https://rvm.io/rvm/install)
 
 ```bash
-rvm install 2.4.2
+CPPFLAGS=-DUSE_FFI_CLOSURE_ALLOC rvm install 2.5.9
```

Comment:
Could we add a comment explaining this flag?

Reply:
@gnclmorais sure, although we are not going to stay on this version of Ruby for long, now that deploy is unblocked we are going to start upgrading Ruby/Rails